### ISSUE TITLE
Filelist generates wrong links when Yellow is installed in a subdirectory

### ISSUE THREAD

**Issue by schulle4u** [GiovanniSalmeri/yellow-filelist#1]
If Yellow is installed in a subdirectory, e.g. `https://example.com/yellow/`, the filelist doesn't recognize the subdirectory and tries to download files from document root. I suggest to add `serverBase` to the path on line 30, this should fix the issue: 

`$fileLoc = $this->yellow->system->get("serverBase").$this->yellow->system->get("filelistLocation").$filePath;`

Steffen

**Comment by GiovanniSalmeri** [comment]
Thank you very much @schulle4u ! Fixed with 92ebbac8d03e068e7d0c60526bcfd9209caf95b5

**Comment by markseuffert** [comment]
Btw, in case someone enters `[filelist ../../system/settings//]` you can do something like:

```php
$filePath = $this->yellow->toolbox->normaliseTokens($filePath);
if ($this->yellow->lookup->isValidFile($filePath) && is_dir($filePath)) {
    //TODO: do more here
}
```
